src/Engines/Node.php

```diff
@@ -27,7 +27,13 @@ public function run(string $script): string
 
         file_put_contents($tempFilePath, $script);
 
-        $process = new Process("{$this->nodePath} {$tempFilePath}");
+        // Version 4.2 introduced fromShellCommandline
+        $command = "{$this->nodePath} {$tempFilePath}";
+        if (method_exists(Process::class, 'fromShellCommandline')) {
+            $process = Process::fromShellCommandline($command);
```

Comment:
What does this doe that the constructor doesn't?

Reply:
The signature of constructor changes in `symfony/process` 5.0. Now it only accepts commands as an array, not a string. It also enforces additional escaping with the array format. Without relying on reflection, I think using `fromShellCommandline` (which takes an unescaped string) is the best way to go.

Reply:
Thanks for the clarification!